applications/users/serializers.py

```diff
@@ -0,0 +1,34 @@
+from rest_framework import serializers
+from dj_rest_auth.registration.serializers import RegisterSerializer
+from dj_rest_auth.serializers import LoginSerializer, UserDetailsSerializer
+
+from applications.users.models import User
+
+
+class UserSerializer(UserDetailsSerializer):
+    class Meta:
+        model = User
+        fields = ['id', 'email', 'name', 'gender']
+
+
+class RestAuthLoginSerializer(LoginSerializer):
+    class Meta:
+        model = User
+        fields = ['email', 'password']
+
+
+class RestAuthRegisterSerializer(RegisterSerializer):
+    email = serializers.EmailField(required=True)
+    password1 = serializers.CharField(write_only=True)
+    gender = serializers.ChoiceField(choices=User.Gender.choices, required=True)
+    name = serializers.CharField(required=True, max_length=250)
```

Comment:
I think you only need to have `gender` here because the others are probably already defined the same way in `RegisterSerializer`. Maybe you can check the code there and see if you can remove the ones that are the same.